stable_baselines3/common/logger.py

```diff
@@ -24,15 +25,43 @@
 
 
 class Video(object):
-    """
-    Video data class storing the video frames and the frame per seconds
-    """
-
     def __init__(self, frames: th.Tensor, fps: Union[float, int]):
+        """
```

Comment:
the docstring was at the right place... (otherwise sphinx does not take it into account...)